packages/vuelidate/index.d.ts

```diff
@@ -40,7 +40,7 @@ export interface ValidatorResponse {
   [key: string]: any
 }
 
-export type ValidatorFn <T = unknown> = (value: T) => boolean | ValidatorResponse;
+export type ValidatorFn <T = unknown> = (value: T) => boolean | Promise<boolean> | ValidatorResponse;
```

Comment:
It can be Promise<ValidatorResponse> as well if not mistaking

Reply:
Done